stravalib/client.py

```diff
@@ -966,15 +1036,22 @@ def get_activity_laps(self, activity_id):
             result_fetcher=result_fetcher,
         )
 
+    # TODO this also has commented code
```

Comment:
this is the other method with commented out code @jsamoocha . just cleaning up a bit :)

Reply:
removed and left a todo to remove the method in a future pr